### PR TITLE
[code] Cleanup unused replace rules

### PR DESCRIPTION
**Pull request by jeanp413**
## Description

<details>
<summary>Summary generated by Copilot</summary>

### <samp>🤖 Generated by Copilot at cc09d2c</samp>

This pull request simplifies the code and configuration of the `codehelper` and `blobserve` components, which are responsible for serving the VS Code web server to the IDE. It removes some unnecessary logic and variables that were used to customize the Open VSX registry URL, which is now handled by `blobserve` alone.

</details>

## How to test

- Check vscode web works, webviews, web extension, gp validate, etc

#### Preview status

<p>Gitpod was successfully deployed to your preview environment.</p>
<ul>
	<li><b>🏷️ Name</b> - jp-managing-skunk</li>
	<li><b>🔗 URL</b> - <a href="https://jp-managing-skunk.preview.gitpod-dev.com/workspaces" target="_blank">jp-managing-skunk.preview.gitpod-dev.com/workspaces</a>.</li>
	<li><b>📚 Documentation</b> - See our <a href="https://www.notion.so/gitpod/6debd359591b43688b52f76329d04010#7c1ce80ab31a41e29eff2735e38eec39" target="_blank">internal documentation</a> for information on how to interact with your preview environment.</li>
	<li><b>📦 Version</b> - jp-managing-skunk-gha.18268</li>
	<li><b>🗒️ Logs</b> - <a href="https://console.cloud.google.com/logs/query;query=jsonPayload.kubernetes.host%3D%22preview-jp-managing-skunk%22%0A%0A--%20Filter%20on%20service:%0A--%20jsonPayload.serviceContext.service%3D%22ws-manager-mk2%22%0A;duration=P1D?project=gitpod-core-dev" target="_blank">GCP Logs Explorer</a></li>
</ul>

## Build Options

<details>
<summary>Build</summary>

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer</summary>

- [ ] analytics=segment
- [ ] with-dedicated-emulation
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment / Integration Tests</summary>

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [ ] /werft with-large-vm
- [x] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`. If enabled, `with-preview` and `with-large-vm` will be enabled.
- [ ] with-monitoring
</details>

/hold

### CODE REVIEW

**Comment by akosyakov** [comment]
Web extensions are still pulling from open-vsx.org, but it is known right?

Screenshot 2023-10-09 at 09 32 22

**Review by akosyakov — APPROVED** [review]
thank you!

File: install/installer/pkg/components/blobserve/configmap.go [hunk 1]
```diff
@@ -44,38 +29,11 @@ func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 				ctx.RepoName(ctx.Config.Repository, ide.CodeIDEImage): {
 					PrePull: []string{},
 					Workdir: "/ide",
```
Comment (akosyakov) [inline install/installer/pkg/components/blobserve/configmap.go:31]: it is the only reason why we keep it?

Comment (jeanp413) [reply install/installer/pkg/components/blobserve/configmap.go:31]: Not sure what you mean? we still need the `InlineStatic` replacements

**Comment by jeanp413** [comment]
> Web extensions are still pulling from open-vsx.org, but it is known right?

yep

File: install/installer/pkg/components/blobserve/configmap.go (hunk 1 above)
Comment (akosyakov) [reply install/installer/pkg/components/blobserve/configmap.go:31]: ah, i see also for supervisor

**Pull request merged**